exercises/C-variables/exercise.js

```diff
@@ -1,3 +1,5 @@
 // Start by creating a variable `greeting`
-
+var greeting="Hello world!";
```

Comment:
Hi luna, you can give some space in this part ( greeting = "Hello World!"). this makes seen the code more clear, other than that good job.